docs/doxygen/demos/jobs_demo.dox

```diff
@@ -0,0 +1,68 @@
+/**
+@page jobs_demo AWS IoT Jobs
+
+This demonstration downloads files from URLs received via AWS IoT Jobs.
+(See https://docs.aws.amazon.com/iot/latest/developerguide/iot-jobs.html for an introduction.)
+
+This demo is intended for Linux platforms with the GCC toolchain, curl,
+and libmosquitto installed.  To build this demo, run make.
+
+To install curl and libmosquitto on a Debian or Ubuntu host, run:
+
+    apt install curl libmosquitto-dev
+
+To build the latest version of libmosquitto with ALPN support, clone
+the repository at https://github.com/eclipse/mosquitto , and follow the
+instructions in compiling.txt within the repo.
+
```

Comment:
Should there be any instructions on creation of a "_Thing_" (or at least mention about that along with an AWS docs link to it)?

Reply:
That's covered in the linked developer guide.  I think adding it again here is overkill.